README.zh.md

```
@@ -1,742 +1,157 @@
-**💾 持久化对话管理**  
-多会话支持与自动持久化。跟踪对话历史、跨应用启动恢复会话、管理多个代理,并与云存储无缝同步——所有这些都由框架的集成存储层处理。
+[简体中文](README.zh.md) | [English](README.md)
 
-**🔌 灵活集成**  
-用于快速开发的高级 API(20-30 行代码即可实现聊天 UI),或在需要最大控制时使用低级 API。适用于 WebSocket/HTTP 后端,支持自定义网络适配器,并可轻松嵌入导航栈、Sheet、抽屉或标签页中。
+搞定 AI 的核心逻辑已经够累了，就别再跟列表视图、键盘高度和气泡渲染较劲了。ChatKit 为您提供了一套**生产级、开箱即用**的原生聊天界面，让您的 AI 应用像系统原生 App 一样丝滑。
 
-**📱 原生性能**  
-纯 iOS/Swift 实现,使用原生 WKWebView 渲染交互式组件。无跨平台妥协——专为 iOS 构建,具有最佳内存使用和流畅的 60fps 滚动,即使有数百条消息也是如此。
+<div align="center">
+  <img src="assets/chatkit-beyond-text-hero.png" width="100%">
```

markdownlint-cli2 (0.18.1)

Images should have alternate text (alt text)

(MD045, no-alt-text)